syngenta_digital_dta/s3/adapter.py

```diff
@@ -238,3 +251,23 @@ def __set_acl(self, public_read):
 
     def __generate_publish_data(self, **kwargs):
         return {'presigned_url': self.create_public_url(**kwargs)}
+
+    def __create_tags(self, **kwargs):
```

Comment:
this function should be named like __construct_tag_set() -> the output is the list to supply the TagSet property for `put_object_tagging()`